tests/datasets/test_benin_cashews.py

```diff
@@ -87,3 +95,18 @@ def test_invalid_bands(self) -> None:
 
         with pytest.raises(ValueError, match="is an invalid band name."):
             BeninSmallHolderCashews(bands=("foo", "bar"))
+
+    def test_plot(self, dataset: BeninSmallHolderCashews) -> None:
```

Comment:
Instead of parametrizing the fixture I would split this into two separate test functions, one that is expected to succeed and one that is expected to raise an error. Otherwise, we'll be running all other tests twice.